diploma-project/components/NewProduct.jsx

```diff
@@ -0,0 +1,77 @@
+import {InputField, Textarea} from "../components";
+import {useState} from "react";
+import {addProduct} from "../services/product";
+import { useUserStore } from "../store/userStorage";
+
+const NewProduct = () => {
+    const { user: user } = useUserStore((state) => ({user: state.user}));
+    const [productTitle, setProductTitle] = useState("");
+    const [productImage, setProductImage] = useState(null);
+    const [productDescription, setProductDescription] = useState("");
+    const [error, setError] = useState(null);
+
+    const onImageChange = () => {
+        const fileInput = document.getElementById('file-input');
```

Comment:
It's usually considered a bad practice to manipulate the DOM directly. Instead, you can use the event object that is passed to the onChange handler. Try something like this:

```
const onImageChange = (event) => {
    setProductImage(event.target.files[0]);
}
```